openghg/standardise/_standardise.py

```diff
@@ -395,8 +395,8 @@ def standardise_footprint(
             function_name="standardise", data=compressed_data, metadata=metadata, file_metadata=file_metadata
         )
 
-        fn_response = call_function(data=to_post)
-        response_content: Dict = fn_response["content"]
+        fn_resolutionponse = call_function(data=to_post)
```

Comment:
I think "res" here got accidentally replaced with "resolution" - same for the line below.